### PR TITLE
Runtime-versions move into `device`: iOS & Native tests

### PR DESCRIPTION
**Pull request by Cawllec**
## Goal

Adds iOS tests, and native code tests targeting both Android and iOS.

## Tests

Passed locally

### CODE REVIEW

File: features/runtime_versions.feature [hunk 1]
```diff
@@ -2,24 +2,42 @@ Feature: Runtime versions are included in all requests
 
     Scenario Outline: Uncaught exception contains runtime versions for Android
         When I launch an <platform> app which has an uncaught exception
+        And I wait for 1 second
```
Comment (fractalwrench) [inline features/runtime_versions.feature]: Is there any way we can avoid this?

File: features/runtime_versions.feature [hunk 2]
```diff
@@ -2,24 +2,42 @@ Feature: Runtime versions are included in all requests
 
     Scenario Outline: Uncaught exception contains runtime versions for Android
```
Comment (fractalwrench) [inline features/runtime_versions.feature]: We'll want to rename all the scenarios that refer to Android as they now run on both Android/iOS

File: features/runtime_versions.feature (hunk 1 above)
Comment (Cawllec) [reply features/runtime_versions.feature]: Not from what I can tell, with no wait it fails the iOS tests.  However these should be removed fairly promptly as this repo is updated to MazeRunner 2.  So not ideal, but temporary

Comment (snmaynard) [reply features/runtime_versions.feature]: With maze 2 we can

**Pull request merged**